src/helpers/mailer.js

```diff
@@ -0,0 +1,46 @@
+import dotenv from 'dotenv';
+import mailer from '@sendgrid/mail';
+
+dotenv.config();
+const sendEmail = async (action, to, token) => {
+  const {
+    SENDGRID_API_KEY, EMAIL_SENDER, APP_URL, NODE_ENV
+  } = process.env;
+
+  mailer.setApiKey(SENDGRID_API_KEY);
+
+  const subjectReset = 'Reset your password - Authors Haven';
+  const contentReset = '<h2>Reset Password</h2><p style="font-size: 1rem;">Please Reset your password by visiting the following link:</p>';
+  const linkReset = `${APP_URL}/reset-password/${token}`;
+
+  if (action === 'reset-password') {
+    const message = {
+      to,
+      from: EMAIL_SENDER,
+      subject: subjectReset,
+      text: 'Authors Haven',
+      html: `<div style="background:#ECF0F1;width:100%;padding:20px 0;">
```

Comment:
create a seperate function that generates the email html template string e.g `generateResetEmailTemplate(contentReset, linkReset)`

Reply:
Worked! but not yet pushed